console/console-init/ui/src/components/MetadataListTable/TableRow.tsx

```diff
@@ -128,13 +131,13 @@ export const TableRow: React.FC<ITableRowProps> = ({ rowData }) => {
           ></DataListItemCells>
         </DataListItemRow>
         {Array.isArray(value) &&
-          value.map((data: any) => (
+          value.map((data: any, index: number) => (
```

Comment:
Please revisit the naming of data everywhere in all the files.

Reply:
MetadataListTable is generic component. that's why used data as common variable. Anyways I used row instead of data here.